pkgs/top-level/all-packages.nix

```diff
@@ -11765,11 +11765,11 @@ with pkgs;
 
   wbox = callPackage ../tools/networking/wbox {};
 
-  webassemblyjs-cli = nodePackages."@webassemblyjs/cli";
-  webassemblyjs-repl = nodePackages."@webassemblyjs/repl";
+  webassemblyjs-cli = nodePackages."@webassemblyjs/cli-1.11.1";
+  webassemblyjs-repl = nodePackages."@webassemblyjs/repl-1.11.1";
```

Comment:
Maybe here? We will forgot it otherwise.

Reply:
I mean we'll also forget it there, imo.